CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 # Unreleased
 
+## [7.0.4] - Tue Sep 25 2018
+### Change
+- Upgrade request and yargs dependency libs regards CEVs. 
```

Comment:
`CEVs` -> `CVEs`

Reply:
I am not sure about it. However, it is fine we can remove the CEV.